modules/admin/src/main/kotlin/jp/co/soramitsu/iroha2/AdminIroha2Client.kt

```diff
@@ -0,0 +1,81 @@
+package jp.co.soramitsu.iroha2
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.ktor.client.call.receive
+import io.ktor.client.request.get
+import io.ktor.client.statement.HttpResponse
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import org.hawkular.agent.prometheus.text.TextPrometheusMetricDataParser
+import java.io.InputStream
+import java.net.URL
+
+open class AdminIroha2Client(
+    peerUrl: URL,
+    open var telemetryUrl: URL = URL(
+        peerUrl.protocol,
+        peerUrl.host,
+        DEFAULT_TELEMETRY_PORT,
+        peerUrl.file
+    ),
+    log: Boolean = false
+) : Iroha2Client(peerUrl, log) {
+
+    open var mapper: ObjectMapper = ObjectMapper()
+
+    /**
+     * Sends metrics request
+     */
+    suspend fun metrics(): Map<String, *> {
+        val response: InputStream = client.value
+            .get<HttpResponse>("$telemetryUrl$METRICS_ENDPOINT")
+            .receive()
+
+        return TextPrometheusMetricDataParser(response)
+            .let { mapper.convertValue(it.parse(), Map::class.java) }
+            .cast()
+    }
+
+    /**
+     * Sends health check request
+     */
+    suspend fun health(): Int {
+        return client.value
+            .get<HttpResponse>("$peerUrl$HEALTH_ENDPOINT")
+            .status.value
+    }
+
+    /**
+     * Sends status check request
+     */
+    suspend fun status(): Map<String, *> {
+        return client.value
+            .get<HttpResponse>("$telemetryUrl$STATUS_ENDPOINT")
+            .receive()
+    }
+
+    /**
+     * Sends value configuration request
+     */
+    suspend fun valueConfig(fieldValue: Collection<String>? = null): Map<String, *> {
+        return config(ConfigurationFieldType.VALUE, fieldValue)
+    }
+
+    /**
+     * Sends docs configuration request
+     */
+    suspend fun docsConfig(fieldValue: Collection<String>? = null): String {
```

Comment:
Let's rename it. AFAIK this returns description of config, so let's rename it to `describeConfig` or smth like that. Also update KDoc as well